scripts/collect_steps.py

```diff
@@ -171,6 +169,40 @@ def extract_intermediate_steps(steps):
                     print(" ".join(args), file=sys.stderr)
                     steps.update(collect_steps(args))
 
+OPRF_BREAKDOWN_KEYS = [256]
+OPRF_USER_CAP = [16, 64, 128]
+OPRF_SECURITY_MODEL = ["semi-honest"]
+OPRF_TRIGGER_VALUE = [7]
```

Comment:
If we are ever testing the OPRF protocol with a single value parameter (i.e., breakdown keys, security model, and trigger value), you don't need to define them as lists nor use `for` loops. Instead of the deep `for` chains, replace them with e.g., `ARGS + [..., "-m", OPRF_SECURITY_MODEL, ...]?

Reply:
originally I was thinking of keeping the array so that we can add different values but for now, will revert to your suggestion. In future if we see a need, we can always add one more for loop